### PR TITLE
add react to dependacies

### PR DESCRIPTION
**Pull request by bradjenn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/inandsza/PlayerVs/issues"
   },
-  "homepage": "https://github.com/inandsza/PlayerVs#readme"
+  "homepage": "https://github.com/inandsza/PlayerVs#readme",
+  "dependencies": {
+    "react": "^0.14.2"
```
Comment (inandsza) [inline package.json]: What about adding node modules to gitignore

Comment (bradjenn) [inline package.json]: ah shit, you're right

Comment (bradjenn) [inline package.json]: actually it's already in there dick

Comment (inandsza) [inline package.json]: k bro

**Comment by inandsza** [comment]
+R

**Pull request merged**